neuvector-prime/aws/suse-llc/payg/usage-instructions.md

```diff
@@ -37,7 +37,9 @@ This will return the OIDC provider's ARN, with the ID at the end.
 To provide the necessary permissions, an IAM policy and an attached role are required. The role is passed as an argument during the *helm* deployment. Download [`neuvector-csp-iam-policy.json`](https://suse-marketplace-assets-public.s3.amazonaws.com/neuvector-prime/neuvector-csp-iam-policy.json) to locally inspect the policy. Create the policy with a *policy name* of your choosing:
 
 ```
-aws iam create-policy --policy-name $POLICY_NAME --policy-document https://suse-marketplace-assets-public.s3.amazonaws.com/neuvector-prime/neuvector-csp-iam-policy.json
+curl https://suse-marketplace-assets-public.s3.amazonaws.com/neuvector-prime/neuvector-csp-iam-policy.json > neuvector-csp-iam-policy.json
```

Comment:
Do we want to add curl to the list at the beginning of pre-reqs?

Reply:
[4a1c3d2](https://github.com/SUSE-Enceladus/marketplace-docs/pull/12/commits/4a1c3d2de85772c0dd308ea77042d51aaafa6b6e)